client/ayon_core/hosts/maya/api/pipeline.py

```diff
@@ -64,6 +65,7 @@
 LOAD_PATH = os.path.join(PLUGINS_DIR, "load")
 CREATE_PATH = os.path.join(PLUGINS_DIR, "create")
 INVENTORY_PATH = os.path.join(PLUGINS_DIR, "inventory")
+TEMPLATE_PLUGINS_PATH = os.path.join(PLUGINS_DIR, "template")
```

Comment:
Call me nerd, can that be `"workfile_build"`? I'm afraid that `"template"` could be misleading name.

Reply:
Changed with https://github.com/ynput/ayon-core/pull/425/commits/c6e61028976c2fc1410cb3bac34e9f17ed51fe83 - let me know if this is what you like.